react-app/src/components/PatientData.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import ReactJson from 'react-json-view';
+import LogList from './LogList';
 
 function PatientData(props) {
   return (
     <div>
-      {props.id === null && (
+      {props.id === null && !props.showLogs && (
         <div className="page-text">
-          <p>Select a patient to view their information.</p>
+          <p>Select a patient to view their information or click on "Log File" to view the logger messages.</p>
         </div>
       )}
-      {props.id !== null && (
+      {props.showLogs && (
+        <div>
+          <h3 className="page-subtitle sticky-item">Log File</h3>
+          <LogList loggedMessages={props.loggedMessages} />
+        </div>
+      )}
+      {props.id !== null && props.id >= 0 && (
```

Comment:
I think this last props.id >= 0 is unnecessary now / could be replaced with `!props.showLogs`